packages/core/types/zapier.generated.d.ts

```diff
@@ -0,0 +1,1746 @@
+/**
+ * This file was automatically generated by Zapier's schema-to-ts tool.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSON Schema
+ * files, and/or the schema-to-ts tool and run its CLI to regenerate
+ * these typings.
+ *
+ * zapier-platform-schema version: 15.10.0
+ *  schema-to-ts compiler version: 0.1.0
+ */
+
+import type { Bundle, ZObject } from './zapier.custom';
```

Comment:
It looks like these are only necessary to support the `PerformFunction`. I wonder, would it make sense to add that `PerformFunction` to the `custom` typings and prevent this import? We'd probably have to eject the entire `Function` object which references `PerformFunction`, but it seems possible from the types dependencies.

Reply:
I see that these are added manually in `transformers.ts`. Are they required to be in this generated declarations file?